examples/qtree_example.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"go-ontap-sdk/ontap"
+	"time"
+)
+
+func main() {
+	// Initialize the ONTAP client
+	client := ontap.NewClient(
+		"http://192.168.200.60",
+		&ontap.ClientOptions{
+			Version:           "1.32", // Ensure this version is compatible with your ONTAP system
+			BasicAuthUser:     "umonitor",
+			BasicAuthPassword: "sxZeJs4n",
+			SSLVerify:         false,
+			Debug:             false, // Enable debugging
+			Timeout:           60 * time.Second,
+		},
+	)
+
+	// Fetch Qtree information iteratively
+	qtreeOptions := &ontap.QtreeListIterOptions{
```

Comment:
this comment refers to all of the places QtreeListIter is used, including the name of the file itself.
the name has to follow the other conventions, it is very important. 
If all of the other place it is used in the code are call xGetIter (x is the name of the netapp data you use), you can't make up your own name and call it QtreeListIter, has to be QtreeGetIter